devel.yaml.jinja

```diff
@@ -20,8 +20,15 @@ services:
     ports:
       - "127.0.0.1:{{ macros.version_major(odoo_version) }}899:6899"
       - "127.0.0.1:{{ macros.version_major(odoo_version) }}069:8069"
+      {%- if odoo_version >= 16 %}
+      - "127.0.0.1:{{ macros.version_major(odoo_version) }}072:8072"
```

Comment:
I don't think this is needed. Development is done in threaded mode, so only 1 port is needed.

Reply:
I agree that most users won't ever need it, but if for some reason someone tries to run `devel.yaml` with `--workers` > 1 (maybe they need to debug something in pre-fork mode), then it won't work.

Still, if you are sure, I will revert it.

Reply:
If that's the case, then don't restrict this to only v16+. Older versions also used that port, right?

Reply:
You're right, fixed it.